pkg/handler/handler.go

```diff
@@ -360,6 +373,8 @@ func (m *Modifier) MutatePod(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResp
 
 	// determine whether to perform mutation
 	if podRole == "" {
+		klog.V(3).Infof("Pod was not mutated. %s",
+			logContext(pod.Name, pod.GenerateName, pod.Spec.ServiceAccountName, pod.Namespace))
```

Comment:
I think it will be more valuable if we log the reason why it's not muted.

Reply:
I will add more logging in the upcoming PR